Sources/MockoloFramework/Models/ParsedEntity.swift

```diff
@@ -66,7 +104,7 @@ final class EntityNodeSubContainer {
 // Contains arguments to annotation
 // Ex. @mockable(typealias: T = Any; U = String; ...)
 struct AnnotationMetadata {
-    var typealiases: [String: String]?
+    var overrides: [String: String]?
```

Comment:
Seems weird to have a struct with one optional param. Shouldn't the whole struct be optional?

Reply:
It might have other things like module name etc down the road